electricitymap/contrib/config/model.py

```diff
@@ -69,7 +69,7 @@ def get_function(self, type: str) -> Optional[Callable]:
         """
         function_str = getattr(self, type)
         if function_str:
-            return import_string(f"electricitymap.contrib.parsers.{function_str}")
```

Comment:
This was just to get the tests running locally, let me know if I should revert it.